docker-compose.yml

```diff
@@ -4,28 +4,32 @@ volumes:
     postgres_data:
 
 services:
-    elasticsearch:
+    elasticsearch2:
         build:
             context: ./
             dockerfile: ./docker/elasticsearch/Dockerfile
         ports:
-            - "9200:9200"
-            - "9300:9300"
-    elasticsearch7:
-        image: elasticsearch:7.4.2
-        ports: 
             - "9000:9200"
             - "9100:9300"
+    elasticsearch:
+        build:
+            context: ./
+            dockerfile: ./docker/elasticsearch/7.4/Dockerfile
+        ports: 
+            - "9200:9200"
+            - "9300:9300"
         environment:
             discovery.type: single-node
     kibana:
-        image: kibana:7.4.2
+        image: blacktop/kibana:7.4
         depends_on:
-            - elasticsearch7
+            - elasticsearch
         ports:
             - "5601:5601"
-        environment:
-            ELASTICSEARCH_HOSTS: http://elasticsearch7:9200
```

Comment:
Does the Kibana container no longer need this variable?

Reply:
I've noticed some flaws in the alpine Kibana image that aren't able to be overcome, one is that it basically only works connecting to `http://elasticsearch:9200` and that is its default setting. This is what lead to the renaming of our ElasticSearch containers and swapping the ports they expose locally.